wd_helper.au3

```diff
@@ -344,7 +344,7 @@ Func _WD_WaitElement($sSession, $sStrategy, $sSelector, $iDelay = Default, $iTim
 				$iErr = $_WD_ERROR_Success
 				ExitLoop
 
-			ElseIf $iErr = $_WD_ERROR_Success Then
+			ElseIf $iErr = $_WD_ERROR_Success And Not $bCheckNoMatch Then
```

Comment:
I'm trying to wrap my head around why this change is needed. Does it actually fix a problem or are you just avoiding the subsequent If statements that should always be False when $bCheckNoMatch is True?

Reply:
when I used this code
```autoit
_WD_WaitElement($sSession, $_WD_LOCATOR_ByXPath, '//body/div[1][@aria-hidden="true"]', 0, 1000 * 60, $_WD_OPTION_NoMatch)
```
I found out that he is leaving immediately, without waiting 1 minutes, even 1 second.

this is because `_WD_FindElement()` was resulting $_WD_ERROR_Success
```autoit
			$sElement = _WD_FindElement($sSession, $sStrategy, $sSelector)
			$iErr = @error

			If $iErr = $_WD_ERROR_NoMatch And $bCheckNoMatch Then
				$iErr = $_WD_ERROR_Success
				ExitLoop

			ElseIf $iErr = $_WD_ERROR_Success Then
```

Reply:
Ok... I see it now. I was initially confused by some of the similar variable names.